src/content_script.js

```diff
@@ -1,6 +1,111 @@
 "use strict";
 
+function itemWidthCheck(target){
+
+  const itemHeight = target.offsetHeight;
+  const itemWidth = target.offsetWidth;
+  let iconClassArr = ["fbc-overlay"];
```

Comment:
Go ahead and use `const` here - you can still call `.push` on it, and it protects us from accidental re-assignment later.

Reply:
Revised!